apidoc/Annotation.yml

```diff
@@ -15,31 +15,31 @@ description: |
     The controls on the left and right side of the annotation can be specified in one of two
     ways:
 
-    * To display an image, set the [leftButton](Titanium.Map.Annotation.leftButton) or 
-      [rightButton](Titanium.Map.Annotation.rightButton) property to an image URL. (On 
-      iOS, you can also use a [SystemButton](Titanium.UI.iPhone.SystemButton) constant 
-      to use one of the native system button icons.) 
```

Comment:
I'm not sure how to do this... I've already deactivated the automatic whitespace on end of lines but it still shows up. Would you recommend me a specific editor?